closeio_api/__init__.py

```diff
@@ -10,6 +10,15 @@
 class APIError(Exception):
     pass
 
+class ValidationError(APIError):
+    def __init__(self, json):
```

Comment:
this could accept `response` instead, use `response.text` for the `super` call and `response.json()` for the lines below. Should be better than converting to json just to convert back to a str, right?